<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgYear</class>
 <widget class="QWidget" name="wdgYear">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>52</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select a year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cmdPrevious">
       <property name="toolTip">
        <string>Previous year</string>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbYear">
       <property name="toolTip">
        <string>Select a year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cmdNext">
       <property name="toolTip">
        <string>Next year</string>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCurrent">
       <property name="toolTip">
        <string>Current year</string>
       </property>
       <property name="text">
        <string>Current Year</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
